src/typescript-service.ts

```diff
@@ -1358,3 +1401,26 @@ function pushall<T>(arr: T[], ...elems: (T | null | undefined)[]): number {
 	}
 	return arr.length;
 }
+
+/**
+ * Maps string-based CompletionEntry::kind to enum-based CompletionItemKind
+ */
+const completionKinds: { [name: string]: CompletionItemKind } = {
```

Comment:
We can't because `CompletionItemKind` is a **const** enum, see https://basarat.gitbooks.io/typescript/content/docs/enums.html#const-enums